sim/rogue/items.go

```diff
@@ -225,3 +225,50 @@ func (rogue *Rogue) ApplyQueensfallRogueEffect(aura *core.Aura) {
 		},
 	})
 }
+
+// Striking a higher level enemy applies a stack of Coup, increasing their damage taken from your next Envenom by 5% per stack, stacking up to 20 times.
+func (rogue *Rogue) ApplyRegicideRogueEffect(aura *core.Aura) {
+	// Coup debuff array
+	debuffAuras := rogue.NewEnemyAuraArray(func(unit *core.Unit, _ int32) *core.Aura {
+		aura := unit.RegisterAura(core.Aura{
+			ActionID:  core.ActionID{SpellID: 1231417},
```

Comment:
I found the debuff the game uses https://www.wowhead.com/classic-ptr/spell=1231424/coup it has a duration of 15s